src/tests/ftest/scrubber/aggregation.py

```diff
@@ -39,12 +39,16 @@ def test_target_eviction_during_aggregation(self):
         self.run_ior_with_pool(create_cont=False)
         telemetry_string = "engine_pool_vos_aggregation_obj_scanned"
         initial_agg_metrics = self.telemetry.get_metrics(telemetry_string)
-        # Enable the aggregation on the pool.
-        self.pool.set_property("reclaim", "time")
         # Now enable the scrubber on the pool.
         self.pool.set_prop(properties="scrub:timed,scrub_freq:1,scrub_thresh:3")
         initial_metrics = self.scrubber.get_scrub_corrupt_metrics()
+        # The disk fault injection is going to be slow.
+        # Reduce transfer size and increase block size for IOR to run for long time.
+        self.ior_cmd.transfer_size.update("1M")
+        self.ior_cmd.block_size.update("20G")
```

Comment:
Can we add these values to the config, please?

Reply:
Sure. How can I implement? 
ior1:
    transfer_size: 5M
    block_size: 250M
ior2:
    transfer_size: 1M
    block_size: 20G.
Any examples?

Reply:
Let me try it ... Thanks.